### ISSUE TITLE
imagenet validation set

### ISSUE THREAD

**Issue by jS5t3r** [haoqiwang/vim#3]
I am not able to reproduce the imagenet validation set. The labels in your list do not match the predictions of my model (model = torch.hub.load('pytorch/vision:v0.10.0', 'wide_resnet50_2', pretrained=True)) at all: 
https://github.com/haoqiwang/vim/blob/master/datalists/imagenet2012_val_list.txt

**Comment by haoqiwang** [comment]
We followed the labels in https://github.com/deeplearning-wisc/large_scale_ood/tree/master/data_lists. It might be different from labels in other repos. It does not affect the OOD evaluation if you use the same version of training and validation labels.
